packages/venia-ui/lib/components/MiniCart/miniCart.css

```diff
@@ -1,79 +1,79 @@
 .root {
-    /* The root aside element takes up the whole screen. */
-    position: fixed;
-    left: 0;
-    top: 0;
     height: 100%;
-    width: 100%;
-
-    /* It is hidden by default. */
+    left: 0;
     opacity: 0;
-    visibility: hidden;
-
-    /* It animates to being closed. */
+    position: fixed;
+    top: 0;
     transition-duration: 192ms;
-    transition-timing-function: var(--venia-global-anim-out);
     transition-property: opacity, visibility;
-
-    /* It sits over all background content. */
+    transition-timing-function: var(--venia-global-anim-out);
+    visibility: hidden;
+    width: 100%;
     z-index: 3;
 }
 
 .root_open {
     composes: root;
-
     opacity: 1;
-    visibility: visible;
-
-    /* It animates to being open. */
     transition-duration: 224ms;
     transition-timing-function: var(--venia-global-anim-in);
-}
-
-.body {
-    overflow-y: scroll;
-    padding: 1.5rem 1rem 1.5rem 0rem;
+    visibility: visible;
 }
 
 .contents {
     background-color: rgb(var(--venia-global-color-background));
     border-radius: 1px;
-    box-shadow: 1px 1px 5px rgb(var(--venia-global-color-gray-darker));
+    box-shadow: 0 1px 6px rgb(var(--venia-global-color-gray-900) / 0.2);
     display: grid;
-    /* The header and footer heights may change based on PWA-116 and PWA-601. */
-    grid-template-rows: 2rem 1fr minmax(2rem, auto);
     /* These heights may change based on PWA-605 and should be based on 1.5 and 2.5 visible items. */
-    min-height: 20rem;
-    max-height: 32rem;
-    overflow: hidden;
-    padding: 1rem;
     position: fixed;
     right: 0.5rem;
     /* This value is equal to the site header height. */
     top: 5rem;
     transform: translate3d(0, -8px, 0);
     transition-duration: 192ms;
-    transition-timing-function: var(--venia-global-anim-out);
     transition-property: transform;
-    /* 360px / 16 root font size = 22.5rem. */
+    transition-timing-function: var(--venia-global-anim-out);
     width: 22.5rem;
 }
 
 .contents_open {
     composes: contents;
-
-    /* It animates "down" from the header. */
     transform: translate3d(0, 0, 0);
     transition-duration: 224ms;
     transition-timing-function: var(--venia-global-anim-in);
 }
 
-.footer {
-    border-top: 2px solid rgb(var(--venia-global-color-gray));
-    overflow: hidden;
+.header {
+    align-items: center;
+    border-bottom: 2px solid rgb(var(--venia-global-color-gray-100));
+    display: grid;
+    font-size: var(--venia-global-typography-heading-XS-fontSize);
+    font-weight: 600;
+    grid-auto-flow: column;
+    grid-template-columns: 1fr;
+    height: 3rem;
+    line-height: var(--venia-global-typography-heading-lineHeight);
+    margin: 0 1rem;
+    padding: 1rem 0.5rem 0.5rem;
 }
 
-.header {
-    border-bottom: 2px solid rgb(var(--venia-global-color-gray));
+.body {
```

Comment:
You alphabetized all of the properties but then moved "body" after "header" ?!

😄